shopify-client/src/components/Header.vue

```diff
@@ -0,0 +1,39 @@
+<template>
+  <div>
+    <b-navbar toggleable="lg" type="light" variant="light">
+
+      <b-navbar-toggle target="nav-collapse"></b-navbar-toggle>
+
+      <b-collapse id="nav-collapse" is-nav>
+        <b-navbar-nav>
+          <b-nav-form>
+            <b-form-input placeholder="Search"></b-form-input>
+          </b-nav-form>
+        </b-navbar-nav>
+      </b-collapse>
+
+        <b-navbar-nav class="ml-auto">
+          <b-navbar-brand to="/app">MiniShopify</b-navbar-brand>
+        </b-navbar-nav>
+
+      <b-collapse id="nav-collapse" is-nav>
+        <!-- Right aligned nav items -->
+        <b-navbar-nav class="ml-auto">
+          <b-nav-item to="/app/about">About</b-nav-item>
+          <b-nav-item to="/app/login">Login</b-nav-item>
+          <b-nav-item to="/app/signup">Sign Up</b-nav-item>
+        </b-navbar-nav>
+
+      </b-collapse>
+    </b-navbar>
+  </div>
```

Comment:
_Nitpick_: unnecessary div (?)

Reply:
it is necessary

Reply:
Yes since we have `<b-navbar>` wrapping everything already.

Reply:
Div left for styling/doc structure, may use later.